### PR TITLE
inline format!() args

### PR DESCRIPTION
**Pull request by matthiaskrgr**
See https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/Opening.20.60E-easy.60.20meta.20issue.28s.29.20for.20small.20tweaks.3F

Edit: this is not complete yet because clippy hans at some point, I'm patching out the responsible lint and will update later

### CODE REVIEW

File: library/core/src/time.rs [hunk 1]
```diff
@@ -1129,10 +1129,10 @@ impl fmt::Debug for Duration {
 
                     // If the user request a precision > 9, we pad '0's at the end.
                     let w = f.precision().unwrap_or(pos);
```
Comment (fbstj) [inline library/core/src/time.rs:1131]:
```suggestion
                    let width = f.precision().unwrap_or(pos);
```
for better legibility?

**Comment by saethlin** [comment]
Can you split this up into multiple PRs? There's no reason we need to land changes to every part of the codebase at the same time.

**Comment by matthiaskrgr** [comment]
@saethlin what do you have in mind? one pr per crate?

Anyway I want to do a all-in-one perf run to see if this has some sneaky sideeffects.

**Comment by matthiaskrgr** [comment]
:thinking:  interesting

File: compiler/rustc_apfloat/src/ieee.rs [hunk 2]
```diff
@@ -570,9 +570,9 @@ impl<S: Semantics> fmt::Display for IeeeFloat<S> {
 
             // Exponent always at least two digits if we do not truncate zeros.
             if truncate_zero {
-                write!(f, "{:+}", exp)?;
+                write!(f, "{exp:+}")?;
             } else {
-                write!(f, "{:+03}", exp)?;
```
Comment (compiler-errors) [inline compiler/rustc_apfloat/src/ieee.rs:575]: I think apfloat changes need to be reverted?

Comment (lcnr) [reply compiler/rustc_apfloat/src/ieee.rs:575]: not sure if that is still the case, cc @eddyb

Comment (lcnr) [reply compiler/rustc_apfloat/src/ieee.rs:575]: but it does make sense to leave this out of this PR, because it at least requires a submodule? update

**Comment by WaffleLapkin** [comment]
r? WaffleLapkin

+1 to @saethlin's proposal of splitting this up.
@matthiaskrgr I think you can bundle multiple crates at a time, just making sure the diff is < 500 lines or something, so it's easier to review changes, rebase the PR when/if needed and fixup nits.

**Comment by Nilstrieb** [comment]
note #105890
I don't think it's worth converting legacy translations, that will just cause conflicts.
so you should probably see which crates/modules now have the denys and only apply those.

**Comment by WaffleLapkin** [comment]
This was splitup into  #114066, #114067, #114068, #114074, #114075, as such — closing.

**Pull request closed without merging**